lisk-dex-electron/src/PlaceOrder.jsx

```diff
@@ -126,42 +129,43 @@ export default class PlaceOrder extends React.Component {
   }
 
   validateOrder() {
+    const baseFee = this.getBaseFee();
     let success = true;
     const { marketOptions } = this.context.configuration.markets[this.context.activeMarket];
     const { priceDecimalPrecision } = marketOptions;
     const sourceAsset = this.props.side === 'bid' ? this.context.activeAssets[1] : this.context.activeAssets[0];
+    const sourceAssetBaseFee = baseFee[sourceAsset];
+    const actualAssetBalance = this.getAssetBalance() - sourceAssetBaseFee;
     const { unitValue } = this.context.configuration.assets[sourceAsset];
     const minOrderAmount = marketOptions.chains[sourceAsset].minOrderAmount / unitValue;
+    const price = parseFloat(this.state.price);
+    const amount = parseFloat(this.state.amount);
     const errors = {
       price: null,
       amount: null,
     };
-    if (!this.state.marketMode) {
-      if (Number.isNaN(this.state.price) || this.state.price === '') {
-        errors.price = 'The order price must be a number.';
-        success = false;
-      } else {
-        const price = parseFloat(this.state.price);
-        if (price === 0) {
-          errors.price = 'The order price cannot be 0.';
-          success = false;
-        } else if (priceDecimalPrecision != null && (this.state.price.toString().split('.')[1] || '').length > priceDecimalPrecision) {
-          errors.price = `The order price for this DEX market cannot have more than ${priceDecimalPrecision} decimal place${priceDecimalPrecision === 1 ? '' : 's'}.`;
-          success = false;
-        }
-      }
-    }
     if (Number.isNaN(this.state.amount) || this.state.amount === '') {
       errors.amount = 'The order amount must be a number.';
       success = false;
-    } else {
-      const amount = parseFloat(this.state.amount);
-      if (amount < minOrderAmount) {
-        errors.amount = `The specified amount was less than the minimum order amount allowed by this DEX market which is ${
-          minOrderAmount
-        } ${sourceAsset.toUpperCase()}.`;
+    } else if (amount > actualAssetBalance) {
```

Comment:
There is something wrong with this logic somewhere because the red error message `The specified amount was less than the minimum order amount allowed by this DEX market which is 10 LSK.` no longer comes up when placing a limit order (it only shows up when placing a market order).

To reproduce, log into a wallet, then place a limit order, type `5` as the amount and submit, the error message does not show up.

Reply:
Seems it was default IDE refactoring, I thought it was an intelligent refactoring - swap of if else condition.
I will fix and push right away